src/AdultEntertainment.php

```diff
@@ -920,20 +1122,6 @@ public function numberOfEmployees($numberOfEmployees)
         return $this->setProperty('numberOfEmployees', $numberOfEmployees);
     }
 
-    /**
```

Comment:
This type is an organization only making offers but not getting offered.
https://schema.org/AdultEntertainment